scripts/Calibration/tofpd/group_calibration.py

```diff
@@ -177,16 +190,16 @@ def cc_calibrate_groups(data_ws,
     DeleteWorkspace('_accum_cc')
     DeleteWorkspace('_tmp_group_cc')
     DeleteWorkspace('_tmp_group_cc_raw')
-    if cycling:
+    if cycling and cycled:
```

Comment:
Could the `cycled` variable be removed and replace this check with
```python
if cycling and '_tmp_group_cc_diffcal' in mtd:
```
to capture the intent?